cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -536,9 +536,9 @@ public static String getGeneratorRegulatingControlMode(Generator generator, Remo
     }
 
     public static String getSvcMode(StaticVarCompensator svc) {
-        if (svc.getRegulationMode().equals(StaticVarCompensator.RegulationMode.VOLTAGE)) {
+        if (svc.getRegulationMode() != null && svc.getRegulationMode().equals(StaticVarCompensator.RegulationMode.VOLTAGE)) {
```

Comment:
```suggestion
        if (svc.getRegulationMode().equals(StaticVarCompensator.RegulationMode.VOLTAGE)) {
```
svc.getRegulationMode should never be null anyway no?